src/extendables/LanguageExtendables.ts

```diff
@@ -1,23 +1,15 @@
-import { LanguageKeysComplex, LanguageKeysSimple } from '@lib/types/Augments';
-import { LanguageKeys } from '@lib/types/Languages';
 import { Extendable, ExtendableStore, Language, LanguageStore } from 'klasa';
 
 export default class extends Extendable {
 	public constructor(store: ExtendableStore, file: string[], directory: string) {
 		super(store, file, directory, { appliesTo: [Language] });
 	}
 
-	public tget<T extends LanguageKeysSimple>(term: T): LanguageKeys[T];
-	public tget<T extends LanguageKeysComplex>(term: T, ...args: Parameters<LanguageKeys[T]>): ReturnType<LanguageKeys[T]>;
-	public tget(this: Language, key: string, ...args: readonly unknown[]): unknown {
-		return this.get(key, ...args);
-	}
-
 	public retrieve(this: Language, key: string) {
-		const value = this.language[key];
+		const value = Reflect.get(this.language, key);
 		if (value) return value;
 
 		const deft = ((this.store as unknown) as LanguageStore).default;
```

Comment:
this can use a better name tbh

Reply:
They're called key-**value** stores for a reason. I think it's fine